tools/ci/docker-containers

```diff
@@ -53,7 +53,7 @@ fi
 
 # Build all of our images.
 
-export BUILD_PLATFORMS=linux/amd64,linux/arm64,linux/ppc64le,linux/s390x
+export BUILD_PLATFORMS=linux/amd64,linux/arm64,linux/arm/v7,linux/ppc64le,linux/s390x
```

Comment:
Did we also want `linux/arm/v6` here?

Reply:
No, we removed linux/arm/v6 from the Docker images a while ago because there's no ARMv6 image variant for our Ubuntu base image.